contributors/design-proposals/node/node-fence.md

```diff
@@ -0,0 +1,528 @@
+# Node fencing
```

Comment:
In general, I'd expect both a proposal and a working prototype to demonstrate the concept.  I consider a fencer as a valuable add-on to most kubernetes clusters, but the proposal needs to address either why every cluster would want it.

Reply:
Basically Huamin started to code the controller behavior in https://github.com/rootfs/node-fencing and I plan to move on with that to demonstrate the concepts better. In pod-safety proposal you refer to everything I write in the overview, here (after the overview) we try to add more detailed flow for executing fencing automatically in cluster. 
It intends to define how we declare different fence phases to isolate and handle node, how we add the configuration to be available in the cluster and more implementation details (as you stated below) to code the Fencing controller and agents executor that are mentioned in the pod-safety design - will add it to the overview and reference to your commit pod-safety.md. 

[1] https://github.com/kubernetes/community/blob/16f88595883a7461010b6708fb0e0bf1b046cf33/contributors/design-proposals/pod-safety.md